pkg/platform/client.go

```diff
@@ -122,6 +124,15 @@ func NewClientFromConfig(ctx context.Context, config *config.CLI) (Client, error
 	return c, nil
 }
 
+// NewClientFromConfig returns a client without the client identity initialized through the selves api.
+// Use this only when performing actions that don't require a log in (like login itself).
+func NewClientFromConfig(config *config.CLI) (Client, error) {
```

Comment:
Why does the function signature contain an error, if the function never returns one?

Reply:
good catch :)